src/alhambra/mixes.py

```diff
@@ -2097,6 +2119,118 @@ def tubes_markdown(self) -> str:
             entries.append(entry)
         return "\n".join(entries)
 
+    def instructions(
+        self,
+        plate_type: PlateType = PlateType.wells96,
+        validate: bool = True,
+        combine_plate_actions: bool = True,
+        well_marker: None | str | Callable[[str], str] = None,
+        title_level: Literal[1, 2, 3, 4, 5, 6] = 2,
+        warn_unsupported_title_format: bool = True,
+        tablefmt: str | TableFormat = "github",
+        floatfmt="g",
+        numalign="default",
+        stralign="default",
+        missingval="",
+        showindex="default",
+        disable_numparse=False,
+        colalign=None,
+    ) -> str:
+        """
+        Returns string combiniing the string results of calling :meth:`Mix.table` and
+        :meth:`Mix.plate_maps` (then calling :meth:`PlateMap.to_table` on each :class:`PlateMap`).
+
+        :param plate_type:
+            96-well or 384-well plate; default is 96-well.
+        :param validate:
+            Ensure volumes make sense.
+        :param combine_plate_actions:
+            If True, then if multiple actions in the Mix take the same volume from the same plate,
+            they will be combined into a single :class:`PlateMap`.
+        :param well_marker:
+            By default the strand's name is put in the relevant plate entry. If `well_marker` is specified
+            and is a string, then that string is put into every well with a strand in the plate map instead.
+            This is useful for printing plate maps that just put,
+            for instance, an `'X'` in the well to pipette (e.g., specify ``well_marker='X'``),
+            e.g., for experimental mixes that use only some strands in the plate.
+            To enable the string to depend on the well position
+            (instead of being the same string in every well), `well_marker` can also be a function
+            that takes as input a string representing the well (such as ``"B3"`` or ``"E11"``),
+            and outputs a string. For example, giving the identity function
+            ``mix.to_table(well_marker=lambda x: x)`` puts the well address itself in the well.
+        :param title_level:
+            The "title" is the first line of the returned string, which contains the plate's name
+            and volume to pipette. The `title_level` controls the size, with 1 being the largest size,
+            (header level 1, e.g., # title in Markdown or <h1>title</h1> in HTML).
+        :param warn_unsupported_title_format:
+            If True, prints a warning if `tablefmt` is a currently unsupported option for the title.
+            The currently supported formats for the title are 'github', 'html', 'unsafehtml', 'rst',
+            'latex', 'latex_raw', 'latex_booktabs', "latex_longtable". If `tablefmt` is another valid
```

Comment:
It would be useful to use Markdown and not warn for "simple", "grid", and "pipe", which are all pandoc-supported formats.